app/packages/operators/src/operators.ts

```diff
@@ -486,17 +487,38 @@ async function executeOperatorAsGenerator(
   return result;
 }
 
-export function resolveOperatorURI(operatorURI) {
+const HASH = "#";
+
+export function resolveOperatorURI(operatorURI, { keepMethod = false } = {}) {
+  if (!keepMethod && operatorURI.includes(HASH))
+    operatorURI = operatorURI.split(HASH)[0];
   if (operatorURI.includes("/")) return operatorURI;
   return `@voxel51/operators/${operatorURI}`;
 }
 
+export function getTargetOperatorMethod(operatorURI) {
+  if (operatorURI && operatorURI.includes(HASH)) {
+    const parts = operatorURI.split(HASH);
+    return parts[1];
+  }
+  return null;
+}
+
+function resolveOperatorURIWithMethod(operatorURI, params) {
```

Comment:
should we add an exception to disallow operator/plugin name with `#`? (edge case, so it could be a future enhancement to prevent unexpected behaviour)